src/Sulu/Bundle/ContentBundle/Preview/PreviewRenderer.php

```diff
@@ -89,6 +89,8 @@ public function render(PageBridge $content, $partial = false)
         // get controller and invoke action
         $request = new Request($query, $request, [], $cookies);
         $request->attributes->set('_controller', $content->getController());
+        $request->attributes->set('webspace', $content->getWebspaceKey());
+        $request->attributes->set('locale', $content->getLanguageCode());
```

Comment:
Do you know why Symfony is prefixing their attributes with `_`? Maybe we should also do that.